distribution/kubernetes/vector-agent/resources.yaml

```diff
@@ -140,7 +140,7 @@ spec:
         app.kubernetes.io/name: vector-agent
         app.kubernetes.io/instance: vector
         vector.dev/exclude: "true"
-    spec:
+    spec:      
```

Comment:
Stray whitespace, here and below?

Reply:
🤔 weird, these are autogenerated - I'll look a bit... but also not super worried 😄

Reply:
Ah, weird, maybe something with the Go templating then.

Reply:
Or from our scripts, either way - good to keep things clean